src/queries.ts

```diff
@@ -15,6 +15,26 @@ query getUserWork($username:String!, $owner:String!, $repo:String!, $sinceIso: D
           title
           createdAt
           url
+          closingIssuesReferences(first: 10) {
+            edges {
+              node {
+                projectItems(first: 10) {
+                  edges {
+                    node {
+                      project {
+                        title
+                      }
+                      fieldValueByName(name: "Epic") {
```

Comment:
Can we make this a configurable field that the user can set? Not every team may use Epics to group issues.